neural_gpu/neural_gpu.py

```diff
@@ -500,8 +500,10 @@ def attention_query(query, attn_v):
             return tf.reduce_sum(encoder_outputs * tf.expand_dims(mask, 2), 1)
 
           with tf.variable_scope("decoder"):
-            def decoder_loop_fn((state, prev_cell_out, _), (cell_inp, cur_tgt)):
+            def decoder_loop_fn(xxx_todo_changeme, xxx_todo_changeme1):
```

Comment:
It looks like your script left some todos here.

Reply:
These are temporary variables which are immediately consumed on the following line.  The names that I could think up for them (`state__prev_cell_out__unused` and `cell_imp__cur_tgt`) were not any more intuitive/ user friendly than the defaults.

Reply:
An alternative could be the names used for invoking `decoder_loop_fn` on line 543: 
`(embedded_dec_inp_proj, gpu_tgt_trans)`

See
https://github.com/Upgrade-to-Python3/models/blob/fbeb6ad6d13daa9708e32469db40a0cf52aadc6d/neural_gpu/neural_gpu.py#L543

ambitious organization name :)